scripts/clang_tidy_wrapper.sh

```diff
@@ -13,5 +13,5 @@ if grep "$file_relative_to_source" $cmake_source_dir/.clang-tidy-ignore > /dev/n
 	echo "clang-tidy: Ignoring $file_relative_to_source"
 	exit 0
 else
-	exec clang-tidy-6.0 -header-filter=$1/src/ $file $@
+	exec `which clang-tidy-7.0 clang-tidy-6.0 clang-tidy | head -n 1` -header-filter=$1/src/ $file $@
```

Comment:
Here, it is irrelevant, but generally, `$()` is nicer because it can be nested.

Reply:
you got 'em mad bash skillz. ;) Better now?